src/target/renesas.c

```diff
@@ -91,6 +91,102 @@ typedef enum {
 	PNR_SERIES_RA6T2 = PNR_SERIES('A', '6', 'T', '2'),
 } renesas_pnr_series_e;
 
+typedef enum {
+	RENESAS_FLASH_MF3,
+	RENESAS_FLASH_MF4,
+	RENESAS_FLASH_RV40,
+} renesas_flash_type_e;
+
+typedef enum {
+	RENESAS_LOCATION_FIXED1,
+	RENESAS_LOCATION_FIXED2,
+	RENESAS_LOCATION_FRT,
+} renesas_pnr_location_e;
+
+typedef struct renesas_family {
```

Comment:
The order things appear in this structure result in a lot of packing fields being inserted - for example after each uint16_t. Please reorder the fields to put the size information after the start information to reduce the Flash footprint it causes. See the `-Wpacked` compiler flag, or clang-tidy's lints for more information.